pkg/connector/tickets.go

```diff
@@ -52,6 +52,9 @@ func (d *Connector) customFieldSchemaToMetaField(field *v2.TicketCustomField) (i
 				return v, nil
 			}
 		}
+		if len(strValue) == 0 {
+			return nil, nil
+		}
```

Comment:
Beware: if the user of `Connector.customFieldSchemaToMetaField` may trigger a panic if he/she decides to dereference the return value 👀

Reply:
@shackra yes, that is true, it is an existing behavior of this function given line 62. Thank you!